src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -478,6 +610,25 @@ class MessageSection extends Component {
     document.body.style.setProperty('background-color', 'white');
   }
 
+  openFullScreen = () => {
+    this.props.actions.openModal({ modalType: 'chatBubble' });
```

Comment:
You can pass fullScreen prop for ChatApp here, no need to make a separate `ChatAppDialog` component.